lib/JavascriptModulesPlugin.js

```diff
@@ -72,6 +73,7 @@ class JavascriptModulesPlugin {
 							filenameTemplate,
 							pathOptions: {
 								noChunkHash: !useChunkHash,
+								contentHashType: "javascript",
```

Comment:
I assume this means other plugins/loaders that don't produce javascript will need their own hashing system.

Reply:
yep, I'll submit a PR to mini-css-extract-plugin where you can see an example how to use it.